packages/fbtee/src/list.tsx

```diff
@@ -8,6 +8,7 @@ export default function list(
   items: ReadonlyArray<string | ReactElement | null | undefined>,
   conjunction: FbtConjunction = 'and',
   delimiter: FbtDelimiter = 'comma',
+  serialComma: boolean = false,
```

Comment:
Could I suggest making this an option object to improve readability and accommodate for future options?

```suggestion
  options?: {serialComma?: boolean}
```

Reply:
Agreed!

Reply:
Done!